### ISSUE TITLE
Token warnings & instructions did not trigger for github token (or travis token)

### ISSUE THREAD

**Issue by michaelpacer** [conda-forge/conda-smithy#441]
I had no `~/.conda-smithy` directory when trying to run `conda smithy rerender`, it gave warnings & instructions for appveyor, circleci, and anaconda cloud. 

It would seem that https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/github.py#L17 should have been hit and printed out instructions, but that never happened. 

Additionally, nothing triggered whatsoever regarding the travis-ci token loading, but when I look at https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/ci_register.py#L59, it makes me think that the only problem is that I didn't have a github token in the first place.

Note, adding tokens for only appveyor, circleci, and anaconda cloud all were sufficient to allow me to successfully run `conda smithy rerender` and push the resulting commit (see https://github.com/conda-forge/nbconvert-feedstock/pull/8/commits/9b4230f3eaf9261ea4510dba71008db8246850cd). The only reason I even figured all this out is because none of my travis builds were successfully initialising.

**Comment by jakirkham** [comment]
Yeah those warnings can be ignored. Tokens are not needed for re-rendering. Maybe we should suppress those warnings during re-rendering?

**Comment by michaelpacer** [comment]
So I believe that tokens are not needed for rerendering, but is it the case that "tokens" are to be associated with the conda-forge organisation and not individual user accounts? Right now the error message read to me as saying that we are to obtain personal user-account level tokens at the relevant services.

To explain the context: 
I figured something might have been wrong with my tokens (in particular) because when I tried to make a PR against https://github.com/conda-forge/nbconvert-feedstock the TravisCI builds never actually *started* (though the request was acknowledged and the queue populated). When I looked at the tokens that were present, there was no travis token (and no github token). It made sense to me that issues with running the travis build might be a problem related to authentication. 

However, whatever I've done seems to have messed up the whole process, and now none of our travis builds are moving forward (even those on another repo branch that should have been unaffected by my actions). Now I can't cancel those builds (related to https://github.com/travis-ci/travis-ci/issues/4867) and they can't move forward.

Regardless, the situation with tokens has confused me to the point where I wouldn't even know what to say on the mailing list regarding them :/.

**Comment by michaelpacer** [comment]
NB: `conda smithy -h` also asks for you to register tokens.

**Comment by jakirkham** [comment]
Sorry it is confusing. 😞 First off let me assure you that you did nothing wrong in that PR AFAICT. Second the warning is definitely confusing and IMHO should not be there at all. For now simply disregard the warning and I'll look into removing it/relocating it to a more appropriate place.

FWIW tokens are mainly used in automation scripts so almost no one needs them. In very rare cases, they are used for administrative purposes so this falls in the domain of core devs.

As for Travis CI, we are seeing some issues with their backlog growing and it is causing us some issues. ( https://github.com/conda-forge/conda-forge.github.io/issues/306 ) This was not caused by your PR though maybe it made a small difference. FWIW I have canceled the Travis CI builds associated with that PR and any old ones.

Ideally we would do this throughout the org, but it is simply not sustainable (i.e. no one has time to find these dups and click cancel 100s of times). We have looked at ways to automate these cancellations ourselves and have asked Travis CI to only allow the latest build to run. Sadly we ran into roadblocks in all previously tried cases. Will try giving the problem another look.
